.gitignore

```diff
@@ -4,3 +4,4 @@ typings/
 /lib/
 /dist/
 /.tmp/
+.idea
```

Comment:
Can you revert this change? This kind of file should be put your local gitignore.